.github/workflows/spell-checker.yml

```
@@ -0,0 +1,24 @@
+name: Spell Checker
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  yaml-format:
```

actionlint (1.7.4)

"steps" section is missing in job "yaml-format"

(syntax-check)